chapters/api_concepts.txt

```diff
@@ -412,11 +412,81 @@ Index>> appendix.
 === Arrays and Memory Ownership
 :array: <<api_concepts_memory, array>>
 
-`ANARIArray` handles represent data arrays in memory, which are shared with
-device implementations. The memory shared with ANARI can be either owned
-by the application, have ownership transferred to the device via a deleter
-callback, or be managed by the device. Input data has dimensions encoded
-in its handle type, where each dimension can have a sparse stride.
+There are two methods of expressing array data on objects: directly mapping
+array elements on an object parameter in order to write data elements, or
+creating array objects represented by a handle.
+
+The first method lets applications directly map a device-internally allocated
```

Comment:
Not sure we need to talk about "allocation" as that sounds a bit prescriptive on the implementation side and you are not supposed to "allocate" user side data this way obviously.
> The first method lets applications directly map a device managed...

Reply:
Addressed in ab76bd4.